commands/plugin/resoure_pack_link.js

```diff
@@ -0,0 +1,17 @@
+const { Command } = require('discord.js-commando');
+
+module.exports = class WikiLink extends Command {
+	constructor(client) {
+		super(client, {
+			name: 'resourcePack',
+			aliases: ['resourcePack'],
+			group: 'wiki',
```

Comment:
```suggestion
			group: 'plugin',
```
The above command is in the plugin folder, either move to wiki folder or change group as above.